src/main/frontend/modules/shortcut/data_helper.cljs

```diff
@@ -93,13 +93,10 @@
 (defn decorate-binding [binding]
   (-> (if (string? binding) binding (str/join "+"  binding))
       (str/replace "mod" (if util/mac? "cmd" "ctrl"))
-      (str/replace "ctrl" (if util/mac? "cmd" "ctrl"))
-      (str/replace "cmd" (if util/mac? "cmd" "ctrl"))
       (str/replace "alt" (if util/mac? "opt" "alt"))
       (str/replace "shift+/" "?")
       (str/replace "left" "←")
       (str/replace "right" "→")
-      (str/replace "+" " ")
```

Comment:
"+" matters for multi-stage shortcuts
like:
ctrl+x ctrl+s

Reply:
if "+" is replaced with " ":
then "ctrl+x s" will be shown as "ctrl x s", which is misleading.